internal/service/ocm/ocm_impl_test.go

```diff
@@ -0,0 +1,59 @@
+package ocm
+
+import (
+	"os"
+	"testing"
+
+	"github.com/redhatinsights/mbop/internal/config"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSuite struct {
+	suite.Suite
+	IsInternalLabel string
+}
+
+func (suite *TestSuite) SetupSuite() {
+	suite.IsInternalLabel = "foo"
+}
+
+func (suite *TestSuite) SetupTest() {
+	config.Reset()
+}
+
+func (suite *TestSuite) TestGetIsInternalMatch() {
+	os.Setenv("IS_INTERNAL_LABEL", "foo")
+	l := &v1.LabelBuilder{}
+	l.Value(suite.IsInternalLabel)
+	acctB := &v1.AccountBuilder{}
+	acctB.Labels(l)
+	acct, _ := acctB.Build()
+	assert.Equal(suite.T(), true, getIsInternal(acct))
```

Comment:
Super minor - but there is actually an `Equal` method right on the suite: `suite.Equal(true, getIsInternal(acct))` if you were interested in using that so you don't have to pass the suite as an argument. 

I don't mind either way since they're the same package - ~I also think suite.Equal calls assert.Equal under the hood~ it looks like its embedded. Fun!